tests/regression/method.c

```diff
@@ -257,20 +289,249 @@ static result_t cli_compress(
     return result_data(data);
 }
 
+static int advanced_config(
+    ZSTD_CCtx* cctx,
```

Comment:
I'm using clang format. I set `IndentWidth` to 4 to match the rest of zstd, but it also has `ContinuationIndentWidth` as 4, which controls all continued lines indents. Setting it to 8 seems too large. Would it make sense to just set `IndentWidth` to 2?

Reply:
It seems like a good idea to use a formatting tool.

`ContinuationIndentWidth` seems to impact more than just function parameters,
but it also seems to be the right value which controls function parameters indentation.

If it was only about function parameters indentation, I would indeed suggest
a larger `ContinuationIndentWidth`.

But since this variable also seems to impact other places in the code,
it's less clear to me what would be the side effects.